src/GovUk.Education.ExploreEducationStatistics.Data.Importer/Services/ImporterService.cs

```diff
@@ -123,16 +127,25 @@ public SubjectMeta GetMeta(DataTable table, Subject subject, StatisticsDbContext
             return _importerMetaService.Get(table.Columns, table.Rows, subject, context);
         }
 
-        public void ImportFiltersLocationsAndSchools(DataColumnCollection cols, DataRowCollection rows, SubjectMeta subjectMeta, StatisticsDbContext context)
+        public async Task ImportFiltersLocationsAndSchools(DataColumnCollection cols, DataRowCollection rows,
+            SubjectMeta subjectMeta, StatisticsDbContext context, Guid releaseId, string origDataFileName)
         {
             // Clearing the caches is required here as the seeder shares the cache with all subjects
             _importerFilterService.ClearCache();
             _importerLocationService.ClearCache();
 
             var headers = CsvUtil.GetColumnValues(cols);
+            var rowCount = 1;
+            var dataRows = rows.Count;
```

Comment:
Probably better to call this `totalRows` to avoid any confusion

Reply:
Renamed to `totalRows`